### PR TITLE
implemented react router

### PR DESCRIPTION
**Pull request by Gumatan**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
ok. I just changed the gamelist file extension

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-      <div className="bigGamePage">
-      <div className="gamePage">
-      {GameList.map ((game) => {
-      return (<GameChoice name={game.name} picture={game.picture} />)})}
-      </div>
-      </div>
-    </div>)
-};
-
+      <Switch>
```
Comment (LoicBrassart) [inline frontend/src/App.js]: I think there was a merge problem here.

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

File: frontend/src/App.js (hunk 1 above)
Comment (ghost) [reply frontend/src/App.js]: Ok. I 've deleted the first switch and added the Navbar to App.js

**Review by ghost — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**